cypress/integration/singlePetPage_spec.js

```diff
@@ -1,27 +1,55 @@
 describe('Pet Applications Component', () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: 'POST',
+        url: 'https://not-fur-gotten-be.herokuapp.com/graphql',
+        headers: {
```

Comment:
Nice job figuring out that the apollo client middleware was set up dynamically, so that we could just switch the operation name whenever we send requests.